src/App.css

```diff
@@ -12,5 +12,13 @@ body {
   flex-direction: column;
   align-items: center;
   justify-content: left;
-  row-gap: 1rem;
 }
+
+.container {
+  height: 100%;
+  display: flex !important;
+  flex-direction: column;
+  align-items: center;
```

Comment:
There is no need to center the flex container's content. It is a list, and its items should go from the beginning of the page and further.